### ISSUE TITLE
5.1 Feedback: Devices > Module Cards > Magnetic module Set engage height 

### ISSUE THREAD

**Issue by y3rsh** [Opentrons/opentrons#10015]
# Overview

On the magnetic module set engage height slide out.
The app is sending the wrong command shape to the API.
`POST /commands` is missing the parameter `moduleId: {id for the specific magnetic module returned from GET /modules}`

> This missing moduleId from the params node may be missing from commands to other modules, I see it also missing on the temperature module to set temperature.

## Steps to reproduce

1. With a robot connected that has an attached magnetic module
2. Navigate to the Devices>Select connected robot
3. Use the kebab menu  on the magnetic module to "Set engage height"
4.  In the slide-out set the engage height to 10

## Current behavior

Engage height is not set, error in console.
POST /commands
```js
{
   "data":{
      "commandType":"magneticModule/engageMagnet",
      "params":{
         "engageHeight":10
      }
   }
}
```
## Expected behavior

Engage height is set.
POST /commands
```js
{
   "data":{
      "commandType":"magneticModule/engageMagnet",
      "params":{
         "moduleId":"MODULE_ID",
         "engageHeight":10
      }
   }
}
```

**Comment by y3rsh** [comment]
I am seeing that the call to /modules is using `opentrons-version: 2` it will need to be `3` or `*` to get the module id to use in the command call.

**Comment by sakibh** [comment]
Closing this ticket. Issue was addressed by #10119
